styles/layouts/LatestNews.module.scss

```diff
@@ -0,0 +1,53 @@
+.latestNewsLayout {
+  @apply grid;
+}
+
+.newsItem {
+  @apply grid grid-cols-5 items-center;
+  @apply m-4 p-4;
+  @apply rounded-xl;
+  @apply duration-300 ease-in-out;
+
+  &:hover {
+    @apply bg-hover-1;
+    @apply shadow-lg;
+    @apply transition-[box-shadow,background-color];
+  }
+
+  .left {
+    @apply col-span-4;
+    @apply text-white;
+    @apply grid grid-cols-1;
+
+    .title {
+      @apply text-xl;
+    }
+
+    .date {
+      @apply text-base;
+    }
+  }
+
+  .right {
+    @apply grid;
+    @apply col-span-1;
+
+    .learnMore {
+      @apply ml-auto;
+    }
+  }
+}
+
+.blackOnGreen {
```

Comment:
I don't know how I did not catch this before, but we should change these style names. we name our colors things like `bg-color-3` so we can easily change the page theme without having incorrect names or needing to update names it probably doesnt make sense to link our style names to them either. maybe just name this `primaryStyle`, `secondaryStyle`? I know its a lot less specific but I think it follows the color naming convention.